python/cuml/ensemble/randomforest.pyx

```diff
@@ -352,29 +354,36 @@ class RandomForestClassifier(Base):
     Parameters
     -----------
 
-    n_estimators : number of trees in the forest. default = 10
+    n_estimators : number of trees in the forest.
+                   Default = 10, dtype is int.
     handle : cuml.Handle
-        If it is None, a new one is created just for this class
+             If it is None, a new one is created just for this class.
     split_algo : The type of algorithm to be used to create the trees.
                  0 for HIST, 1 for GLOBAL_QUANTILE and 3 for SPLIT_ALGO_END.
-                 default = 0
+                 Default = 0, dtype is int.
```

Comment:
Most of the other algorithms have a standard way of stating the default and the type that is more consistent and I think it would be a good idea to adapt the same format here. See https://github.com/rapidsai/cuml/blob/a19ae84aa33c0bdc8acb7f5ad1213d9501fedde2/python/cuml/linear_model/linear_regression.pyx#L140 for a good example.

Reply:
done!